src/history_manager.py

```diff
@@ -21,14 +51,15 @@ def get_saved(self):
 		# in all situations around a saving
 		return self._is_saved
 
+	def get_initial_operation(self):
+		return self._undo_history[-1].initial_state
+
+
 	def empty_history(self):
 		"""Probably useless way to explicitely 'forget' the objects. It doesn't
 		really free the memory, but it kinda helps i suppose."""
-		for op in self._undo_history:
-			self._delete_operation(op)
```

Comment:
then you should remove my obsolete string comment, as well as the entire `_delete_operation` method